erpnext/patches/v13_0/move_compliance_item_to_item.py

```diff
@@ -0,0 +1,34 @@
+import frappe
+from frappe.modules.utils import sync_customizations
+
+
+def execute():
+	compliance_items = frappe.get_all("Compliance Item", fields=["*"])
+
+	frappe.reload_doc("stock", "doctype", "item")
+	sync_customizations("bloomstack_core")
+
+	fields = ["requires_lab_tests", "enable_cultivation_tax",
+		"strain_type", "item_category", "enable_metrc",
+		"metrc_id", "metrc_item_category", "metrc_unit_value",
+		"metrc_uom", "metrc_unit_uom", "bloomtrace_id"]
+
+	for item in compliance_items:
+		if not item.item_code:
+			continue
+
+		frappe.db.set_value("Item", item.item_code, "is_compliance_item", True, update_modified=False)
+
+		for field in fields:
+			if item.get(field):
+				frappe.db.set_value("Item", item.item_code, field, item.get(field), update_modified=False)
```

Comment:
-    Set_value will be very expensive since it'll be triggered multiple times just to edit one row, which might take a considerable amount of time.
-    Instead do a get_doc and then update it.

Reply:
This is just a patch, and I don't want to add a timeline entry for these values. It's fine if it takes longer. It'll just run once.

Reply:
> This is just a patch, and I don't want to add a timeline entry for these values. It's fine if it takes longer. It'll just run once.

Makes sense.